rzup/README.md

```diff
@@ -4,9 +4,8 @@ Install, update, or revert to a specific RISC Zero version.
 
 ## Install
 
-<!-- TODO: Replace this friendly redirect URL once set up -->
 ```sh
-curl -L https://risc0-artifacts.s3.us-west-2.amazonaws.com/rzup/install | bash
+curl -L https://raw.githubusercontent.com/risc0/risc0/main/rzup/install | bash
```

Comment:
Not sure if there was a plan for some other link, but IMHO github should be used over aws.

Reply:
We will likely setup a more permanent link, but it will be hosted in a bucket rather than on GitHub.